### DISCUSSION TITLE
在SFTP里编辑文件时，打开文件无法显示全部里面内容

### DISCUSSION THREAD

**Discussion by viyv** [lollipopkit/flutter_server_box#58] [Ideas]
在SFTP里编辑文件时，打开文件无法显示全部里面内容，超出界面部分，不知道怎么移动去编辑，是否能添加一个功能让每一行文字或者代码，超出界面时自动换行，这样更方便编辑

**Comment by lollipopkit** [comment]
https://github.com/lollipopkit/flutter_server_box/assets/10864310/18cb935b-a877-40df-badf-5bd80ef6b562
